### PR TITLE
added Librarian, LibraryBranch, LoanRecord, Publisher, User entities.

### PR DESCRIPTION
**Pull request by abdoulbachir**
added Librarian, LibraryBranch, LoanRecord, Publisher, User entities.

### CODE REVIEW

File: lms-core/src/main/java/com/harrisburgu/lms/entity/Librarian.java [hunk 1]
```diff
@@ -1,4 +1,135 @@
 package com.harrisburgu.lms.entity;
 
 public class Librarian {
+    private Integer id;
+    private String  name;
+    private String phoneNumber;
+    private String address;
+    private String ssn;
+    private String emergencyContact;
+    private Integer libraryBranchID;
```
Comment (AungKyaw-Min) [inline lms-core/src/main/java/com/harrisburgu/lms/entity/Librarian.java:10]: libraryBranchId plz

File: lms-core/src/main/java/com/harrisburgu/lms/entity/LoanRecord.java [hunk 2]
```diff
@@ -1,4 +1,98 @@
 package com.harrisburgu.lms.entity;
 
+import java.time.LocalDateTime;
+import java.util.Objects;
+
 public class LoanRecord {
+    private Integer userID;
+    private Integer libraryBranchId;
+    private Integer bookID;
```
Comment (AungKyaw-Min) [inline lms-core/src/main/java/com/harrisburgu/lms/entity/LoanRecord.java:9]: bookId plz

File: lms-core/src/main/java/com/harrisburgu/lms/entity/Publisher.java [hunk 3]
```diff
@@ -1,4 +1,138 @@
...
+import java.util.Objects;
+
 public class Publisher {
+    private Integer id;
+    private String name;
+    private String address;
+    private String phoneNumber;
+    private String email;
+    private String type;
+    private Date establishmentDate;
```
Comment (AungKyaw-Min) [inline lms-core/src/main/java/com/harrisburgu/lms/entity/Publisher.java:13]: You should use localDate

Comment (abdoulbachir) [reply lms-core/src/main/java/com/harrisburgu/lms/entity/Publisher.java]: Done!

File: lms-core/src/main/java/com/harrisburgu/lms/entity/LoanRecord.java (hunk 2 above)
Comment (abdoulbachir) [reply lms-core/src/main/java/com/harrisburgu/lms/entity/LoanRecord.java]: Done!

File: lms-core/src/main/java/com/harrisburgu/lms/entity/Librarian.java (hunk 1 above)
Comment (abdoulbachir) [reply lms-core/src/main/java/com/harrisburgu/lms/entity/Librarian.java]: Done!

**Review by fix201 — APPROVED** [review]
Good job

**Pull request merged**